src/compas-services/Websockets.ts

```diff
@@ -1,5 +1,5 @@
 import { newPendingStateEvent } from '../foundation.js';
-import { createLogEvent, parseXml } from './foundation.js';
+import { APPLICATION_ERROR, createLogEvent, extractErrorMessage, parseXml, processErrorMessage } from './foundation.js';
```

Comment:
`processErrorMessage` is imported but not used

Reply:
Fixed